typescript/cli/src/commands/chains.ts

```diff
@@ -41,13 +55,22 @@ const listCommand: CommandModule = {
         };
         return result;
       }, {});
+    const logMainnet = () => {
+      logBlue('\nHyperlane core mainnet chains:');
+      logGray('------------------------------');
+      logTable(serializer(Mainnets));
+    };
+    const logTestnet = () => {
+      logBlue('\nHyperlane core testnet chains:');
+      logGray('------------------------------');
+      logTable(serializer(Testnets));
+    };
+
+    if (mainnet) return logMainnet();
+    else if (testnet) return logTestnet();
```

Comment:
This is fine, but it would be more direct to simply run the code under the if clause instead of defining an arrow function above and then using it immediately.

Reply:
Since the flags are mutually exclusive and optional, I have the same functions executed below (see line 72 & 73) in order to print out both envs in the absence of both flags. This code reuse was my reasoning for the lambdas

Reply:
Ah, I didn't notice that. Okay agreed